react/features/mobile/navigation/components/conference/components/ConferenceNavigationContainer.js

```diff
@@ -1,157 +1,66 @@
 import { NavigationContainer } from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack';
-import React from 'react';
-import { useTranslation } from 'react-i18next';
-import { useSelector } from 'react-redux';
+import React, { useCallback, useState } from 'react';
 
-import { Chat } from '../../../../../chat';
-import Conference from '../../../../../conference/components/native/Conference';
-import { getDisablePolls } from '../../../../../conference/functions';
-import { SharedDocument } from '../../../../../etherpad';
-import { GifsMenu } from '../../../../../gifs/components';
-import AddPeopleDialog
-    from '../../../../../invite/components/add-people-dialog/native/AddPeopleDialog';
-import LobbyScreen from '../../../../../lobby/components/native/LobbyScreen';
-import { ParticipantsPane } from '../../../../../participants-pane/components/native';
-import { StartLiveStreamDialog } from '../../../../../recording';
-import { StartRecordingDialog }
-    from '../../../../../recording/components/Recording/native';
-import SalesforceLinkDialog
-    from '../../../../../salesforce/components/native/SalesforceLinkDialog';
-import SecurityDialog
-    from '../../../../../security/components/security-dialog/native/SecurityDialog';
-import SpeakerStats
-    from '../../../../../speaker-stats/components/native/SpeakerStats';
+import ConferenceTab from '../../../../../conference/components/native/ConferenceTab';
+import CarmodeTab from '../../../../../conference/components/native/carmode/Conference';
 import { screen } from '../../../routes';
-import {
-    chatScreenOptions,
-    conferenceScreenOptions,
-    gifsMenuOptions,
-    inviteScreenOptions,
-    liveStreamScreenOptions,
-    lobbyScreenOptions,
-    navigationContainerTheme,
-    participantsScreenOptions,
-    recordingScreenOptions,
-    salesforceScreenOptions,
-    securityScreenOptions,
-    sharedDocumentScreenOptions,
-    speakerStatsScreenOptions
-} from '../../../screenOptions';
-import ChatAndPollsNavigationContainer
-    from '../../chat/components/ChatAndPollsNavigationContainer';
+import { navigationContainerTheme } from '../../../screenOptions';
 import {
     conferenceNavigationRef
 } from '../ConferenceNavigationContainerRef';
 
-const ConferenceStack = createStackNavigator();
+import NavigationThumb from './NavigationThumb';
+import styles from './styles';
 
+const ConferenceTabs = createMaterialTopTabNavigator();
 
 const ConferenceNavigationContainer = () => {
-    const isPollsDisabled = useSelector(getDisablePolls);
-    let ChatScreen;
-    let chatScreenName;
-    let chatTitleString;
+    const [ selectedThumb, setSelectedThumb ] = useState(1);
+
+    /**
+     * Lights up the correct bottom navigation circle
+     * in regards with the focused screen.
+     */
+    const onFocused = useCallback(selected => {
+        if (selected === screen.car) {
+            setSelectedThumb(2);
```

Comment:
Can you create an enum instead of using plain numbers?

Reply:
hmm not sure I can do that with current implementation, as `NavigationThumb` component is dynamic, in the sense that it can have as many thumbs as we want. Think I should hard code to 2 thumbs and switch to enum here?

Reply:
I'd say we do that yeah since this is not a generic component we are exporting but a specific usage.